pkg/sql/logictest/testdata/logic_test/cast

```diff
@@ -777,3 +777,26 @@ query T
 EXECUTE s73450_c('foo')
 ----
 f
+
+# Regression test for #74577. Casts between integer types and VARBIT should not
```

Comment:
this probably needs a deprecation process as well, unfortunately.
(detect any usages, and block migration...)

Reply:
I'd like to just fix it before the release. My attempt: https://github.com/cockroachdb/cockroach/pull/74583